### PR TITLE
Lottery system

### PR DESCRIPTION
**Pull request by OffensiveCheese**
Some people will have more power than others, and that's OK. Depends how lucky you are.

### CODE REVIEW

**Comment by mikesmiffy128** [comment]
I'm no python expert, but I'd guess that the RNG is probably exploitable if it's not meant for crypto stuff. Also, this is wrong on principle anyway.

**Comment by OffensiveCheese** [comment]
It's automatically seeded with system time. should be fine

**Comment by mikesmiffy128** [comment]
Seeding with system time is like the worst you can do! :/

**Comment by OffensiveCheese** [comment]
That's apart of the fun

**Comment by PlasmaPower** [comment]
:-1: whenever people make RNG PRs, they forget to look at the mistakes of the past. This means pretty much every PR will be merged.

File: github_api/voting.py [hunk 1]
```diff
@@ -137,6 +138,9 @@ def get_vote_sum(api, votes):
     negative = 0
     for user, vote in votes.items():
         weight = get_vote_weight(api, user)
+        if random.randint(0,500) == 42:
+            #Jackpot!
+            weight = weight * 100
```
Comment (mdcfe) [inline github_api/voting.py]: `weight * 100` is a drastic increase - if this is going to be implemented, it should be much lower, like `* 1.1`, or `* 1.5` at the most.

Comment (mdcfe) [reply github_api/voting.py]: Also, the bot should include in the status check whether anyone hit the jackpot.

Comment (mdcfe) [reply github_api/voting.py]: Correct me if I'm wrong, but isn't this recalculated every time the votes are polled anyway?

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: github_api/voting.py (hunk 1 above)
Comment (rudehn) [reply github_api/voting.py]: yup. This idea has been proposed a lot in the past

**Review by nstanard — CHANGES_REQUESTED** [review]
👿

**Pull request closed without merging**